e2e/tests/room/02-room.spec.js

```diff
@@ -354,6 +361,26 @@ describe('Room screen', () => {
 			});
 		});
 
+		describe('Mark message as unread', async() => {
```

Comment:
Can you move it away from this file?
We have the ability to mark channels are unread as well, so a file for `mark as unread` as a feature would be better.